.ado/azure-pipelines.yml

```diff
@@ -36,43 +36,39 @@ jobs:
           yarn check-for-changed-files
         displayName: 'verify API and Ensure Changed Files'
 
-  # Windows bundling and end to end testing
-  - job: WindowsPR
-    displayName: Windows PR
+  - job: AndroidPR
+    displayName: Android PR
     pool:
-      vmImage: 'windows-2019'
+      vmImage: 'macos-11'
```

Comment:
why we using macos11? so old and it might not have all the latest security stuff.

Reply:
This is probably a relic from pre EO work. Can update.

Reply:
resolved!

Reply:
Turns out the Android CI hangs without this. Bringing this back and I'll file an issue for later :/